source/game/field/CourseColMgr.cc

```diff
@@ -0,0 +1,135 @@
+#include "CourseColMgr.hh"
+
+#include "game/field/CollisionDirector.hh"
+
+#include "game/system/ResourceManager.hh"
+
+// Credit: em-eight/mkw
+
+namespace Field {
+
+void CourseColMgr::init() {
+    void *file = LoadFile("course.kcl");
+    m_gndCol = new KColData(file);
```

Comment:
Subsequent changes because of this should also be included in resolution.

```suggestion
    m_data = new KColData(file);
```